### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by Danwhy**
Improves guide by adding sections on:

- How to get updated items from the database
- Getting "all" (current) items from database
- How to exclude "deleted" items

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -658,6 +660,9 @@ defmodule Append.AppendOnlyLog do
       def get(id) do
       end
 
+      def all() do
```
Comment (nelsonic) [inline README.md]: is the `def all() do` on line 466 (_above_) different from this one?

File: README.md [hunk 2]
```diff
@@ -425,16 +425,17 @@ defmodule Append.AppendOnlyLog do
 
   @callback insert
   @callback get
-  @callback get_by
+  @callback all
```
[the remark this answers is not in the document]
Comment (Danwhy) [reply README.md]: The initial implementation of it didn't actually work, and I feel like it's not a 100% necessary function, so didn't want to add too much unnecessary complexity to the tutorial

File: README.md (hunk 1 above)
Comment (Danwhy) [reply README.md]: No it's the same one, just here as a placeholder until we show how to implement it on line 740

Comment (nelsonic) [reply README.md]: So we can _remove_ it on the next iteration? 🤔

**Pull request merged**